pxtlib/docsrender.ts

```diff
@@ -319,7 +319,7 @@ namespace pxt.docs {
 
         // Add sidebar toggle
         const sidebarToggleHtml = `
```

Comment:
This change would require a cloud update.
We should inline all these macros in the HTML, and eventually get rid of those in docsrender.